### DISCUSSION TITLE
How to add height/width without resizing?

### DISCUSSION THREAD

**Discussion by stamminator** [NickeManarin/ScreenToGif#1135] [Q&A]
A pretty common use case for gifs if having a horizontal bar along the top which contains a text caption. How do I add space to the top of my frames so that I can add in a white box and some caption text? If I use the Resize function, it stretches the image, which is not the intended effect in this use case. Thanks for making this great tool!

**Comment by NickeManarin** [comment]
Hi, right now you can use the Border feature and set a negative value.

For v3, the app will support changing the canvas size and repositioning the recording and other elements.

**Reply by stamminator** [reply]
That does the trick! Thank you
